rose/server/score_test.py

```diff
@@ -43,12 +43,12 @@ def assert_move_left(self):
     def assert_move_back(self):
         assert self.player.x == self.x
         assert self.player.y == self.y + 1
-        assert self.player.score == self.score
+        assert self.player.score == self.score - config.score_move_forward
```

Comment:
I've seen in the test, can someone get a <0 score?
like -10?
I think we should stay at 0 in that case.
maybe use something like `max(self.score - config.score_move_forward, 0)` for avoiding a negative score

Reply:
Each move you get 10 points for moving forward, then your score is modified according to the action you chose.
The first 5 steps are clear of obstacles so you can say that the first obstacle is reached with 60 points.
In those conditions, I suspect that a score lower than 0 won't happen.

Reply:
Even a no action driver finishes the race with a score higher than 250.

Reply:
I was wondering about those due to the failures I got in my patch:
https://travis-ci.org/github/RedHat-Israel/ROSE/builds/744630508
(After solving the flake8 issue and start running the tests)